vernac/comTactic.ml

```diff
@@ -42,9 +42,43 @@ let { Goptions.get = print_info_trace } =
     ~value:None
     ()
 
-let solve_core ~pstate n ~info t ~with_end_tac:b =
+let warn_end_tac =
+  CWarnings.create_warning ~name:"deprecated-end-tac" ~from:[Deprecation.Version.v9_2] ()
+
+let pp_warn_end_tac =
+  CWarnings.create_in warn_end_tac Pp.(fun pptac ->
+      (* XXX we need to print eagerly for the quickfix but it should be delayed *)
+      fmt "Using \"...\" is deprecated, use \"%t\" instead" (fun () -> pptac))
+
+let warn_end_tac ?loc end_tac =
+  match end_tac with
+  | None -> CErrors.user_err ?loc Pp.(str "This \"...\" is useless, use \".\" instead.")
```

Comment:
could make this a warning too, but there doesn't seem to be any point

Reply:
I'm making overlays

Reply:
That's another option (I'd have a slight preference for the warning, but you're the author)